may-change-logs

kghamilton89 opened the pull request:
Omitted issues: 5283, 5312, 5273, 2346 (charts)

rodionos commented:
List of issues for May changelog is specified in 5344 in redmine. Please use it as source.

kghamilton89 commented:
I guess I must have reordered the chart in RedMine before I exported. All issues were there for ATSD but incorrectly ordered.

rodionos commented on changelogs/2018_05/README.md:
```diff
@@ -0,0 +1,50 @@
+5315 | message        | Bug     | Fix incorrect [wildcard](../../search/entity-search.md#wildcards) match on the message search page.
+5313 | client         | Bug     | [Python Client](https://github.com/axibase/atsd-api-python): [Message model](https://github.com/axibase/atsd-api-python#inserting-data) standardized.
+5310 | message        | Feature | Modify filter processing for tag wildcards and empty tag values on the message search page.
+5309 | api-rest       | Bug     | `NullPointerException` raised on [message insert](../../api/network/message.md#message-command) when one of the tags is set to `null`.
```
Link should be set to rest api, not network api

kghamilton89 replied:
REST API linked.

rodionos commented on changelogs/2018_05/README.md:
```diff
@@ -0,0 +1,50 @@
+5338 | export             | Bug     | Series tags validation error on the export page.
+5337 | client | Feature | Value filter added for [Python Client](../../README.md#axibase-time-series-database-client-for-python).
+5334 | api-rest       | Bug     | Properties [delete query](../../administration/data_retention.md#deleting-properties) returns zero counter even after deleting some records.
+5333 | client         | Bug     | [Python Client](../../README.md#axibase-time-series-database-client-for-python) tests are failing due to time precision.
```
https://github.com/axibase/atsd-api-python - correct link to python client

kghamilton89 replied:
Sorry this was a search and replace error

rodionos commented on changelogs/2018_05/README.md:
```diff
@@ -0,0 +1,50 @@
+-----|-------------|---------|----------------------
+5340 | statistics     | Bug     | Server error raised when accessing statistics page for series with [NaN](../../sql/README.md#not-a-number-nan) values.
+5338 | export             | Bug     | Series tags validation error on the export page.
+5337 | client | Feature | Value filter added for [Python Client](../../README.md#axibase-time-series-database-client-for-python).
```
Link to python client: https://github.com/axibase/atsd-api-python

kghamilton89 replied:
likewise

kghamilton89 commented:
Grammar was fine, articles were correct. Generally in something like a Change Log, it's perfectly acceptable to omit most articles, it's a type of shorthand. I typically do it, so I went through and removed articles from where maybe you or I had added them for standardization purposes. 

For reference the [Chrome Release Notes](https://chromereleases.googleblog.com/) almost never contain articles, even where they might be required should the context be more standard. Same is true in [Linux Change Logs](https://kernelnewbies.org/LinuxChanges#Linux_4.15.File_systems).

Articles are just a nightmare (even the sentence I used above could easily be written, "the grammar was fine, the articles were correct."
